api/src/services/artifact-service.ts

```diff
@@ -84,7 +84,11 @@ export class ArtifactService extends DBService {
     });
 
     // Create a new submission for the artifact collection
-    const { submission_id } = await this.submissionService.insertSubmissionRecordWithPotentialConflict(dataPackageId);
+    const { submission_id } = await this.submissionService.insertSubmissionRecordWithPotentialConflict(
+      dataPackageId,
+      'TODO_Temp',
```

Comment:
Is this function being used? could we delete the entire function?

Reply:
There is a lot of old code we can remove. We were just holding off so as no to make actual feature PRs bigger/more complex.

Reply:
That being said, removing small bits as needed if it is hindering progress on current work, is totally fine.